### ISSUE TITLE
Cannot verify grsecuritry integrity

### ISSUE THREAD

**Issue by cthbleachbit** [rickard2/grsecurity-Debian-Installer#20]
```
==> Installing grsecurity stable version 3.1 using kernel version 3.14.51 ... 
==> Installing packages needed for building the kernel ... OK
==> Verifying linux-3.14.51.tar ... OK
==> Verifying grsecurity-3.1-3.14.51-201509112212.patch ... Failed
```

And when i tried to get this file manually by 
`curl --progress-bar --remote-name --tlsv1 --proto =https https://grsecurity.net/stable/grsecurity-3.1-3.14.51-201509112212.patch`
it actually returned a 404 not found error.

**Comment by cthbleachbit** [comment]
This error might be related to [Important Notice Regarding Public Availability of Stable Patches](https://grsecurity.net/announce.php) since I got a 403 forbidden on `https://grsecurity.net/stable/`

**Comment by trueshanti** [comment]
for those having access to the customer-repo it would be fine to adopt this scripts to be available to stay able to use it .. i might submit a patch

**Comment by Emasoft** [comment]
I need to install grsec on a debian machine. Is this still an issue?

**Comment by cthbleachbit** [comment]
Yes, as is stated on grsecurity official website they only releases testing
builds to the public now
